dubbo/src/lib.rs

```diff
@@ -22,6 +22,7 @@ mod framework;
 pub mod invocation;
 pub mod protocol;
 pub mod registry;
+pub mod rpc;
```

Comment:
the meaning of rpc name is too big. I suggest that you can use context directly.